test/test_tree_1d_euler.jl

```diff
@@ -219,7 +219,7 @@ end
     end
 end
 
-@trixi_testset "elixir_euler_ec.jl with flux_hll" begin
+@trixi_testset "elixir_euler_ec.jl with FluxHLL(min_max_speed_naive)" begin
```

Comment:
I'm not sure whether we really want to test the naive version here or the Davis version.

Reply:
Agreed, I use the new default flux here.